public/js/utils/sources-tree.js

```diff
@@ -5,8 +5,8 @@ const { isPretty } = require("../utils/source");
 const IGNORED_URLS = ["debugger eval code", "XStringBundle"];
 
 function isHiddenSource(source) {
-  const url = getURL(source);
-  return !url || url.path.match(/SOURCE/) || IGNORED_URLS.includes(url);
+  const url = source.get("url");
+  return !url || url.match(/SOURCE/) || IGNORED_URLS.includes(url);
```

Comment:
using `getURL` broke down with `SOURCE:formatted`. `getURL` thought source was a host because of the colon. The easy fix was to not use getURL because we don't need paths for this